src/app.md

```diff
@@ -271,6 +272,40 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot]: #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Traverse directories while skipping dotfiles
+
+Iterates over filesystem entries efficiently skipping hidden files and directories with help of [WalkDir::filter_entry]. 
```

Comment:
mm for me it is working because it links to https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.filter_entry with 

```markdown
[`WalkDir::filter_entry`]: https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.filter_entry
```